packages/flutter_tools/templates/module/android/app.tmpl/build.gradle.tmpl

```diff
@@ -0,0 +1,35 @@
+def flutterPluginVersion = 'managed'
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
```

Comment:
I have a thought here that doesn't need to be addressed now....but should details like SDK versions be held in a top level Flutter configuration file that is then also used to describe the min, max, and current SDK versions that Flutter, itself, supports?  In other words, having these values hard coded here looks like it creates an opportunity for the official Flutter supported SDK versions to get out of sync with these templates if someone isn't careful to keep them in sync.

Reply:
I agree, it could make good sense to centralize this information insofar as it reflects what Flutter supports. I do think that we at some point want to allow the app author to specify Android versioning info in their `pubspec.yaml` file, and thus have Flutter tooling generate an Android module and/or ephemeral app configured using the wanted compileSdkVersion/build tool version/support lib version.